grid/address/clean-address/maps/clean-address.ts

```diff
@@ -10,11 +12,13 @@ export const cleanAddressTest = (
     let superface: SuperfaceTest;
 
     beforeEach(() => {
-      superface = new SuperfaceTest({
-        profile: 'address/clean-address',
-        provider,
-        testInstance: expect,
-      });
+      superface = new SuperfaceTest(
+        {
+          profile: 'address/clean-address',
+          provider,
+        },
+        nockConfig
+      );
```

Comment:
Silly question / nitpick: would be worth to create test helper containing factory function to create `SuperfaceTest` and add it to `globals` using https://jestjs.io/docs/configuration/#globals-object?

Reply:
It would definitely make sense now with `nockConfig` 👍🏻 I can try it on one test and let you know.

Reply:
@freaz I had to add it with `setupFiles` since docs says that passing functions to `global` is only possible with `setupFiles` option, but typescript doesn't cooperate, see [this commit](https://github.com/superfaceai/station/pull/279/commits/e12442ac1512d2cc5f41125342406d7da4907507).